showcase/src/app/app.component.ts

```diff
@@ -68,6 +70,14 @@ export class AppComponent implements OnInit {
 		this.breakpointObserver.observe([this.mediaQueryMdSm]).subscribe((state: BreakpointState) => {
 			this.isMenuModeActive = state.matches;
 		});
+
+		this.routingService.addTransitionHook(StarkRoutingTransitionHook.ON_SUCCESS, { to: "**" }, (tran: Transition) => {
```

Comment:
The "init" and "exit" states should be excluded here cause there is no reason to do this logic in those.

Something like this:
https://github.com/NationalBankBelgium/stark/blob/master/packages/stark-core/src/modules/session/services/session.service.ts#L150L162

Reply:
I'm not sure what you are asking for here. 😕 This needs to be run on every state change right?

Reply:
Well, that's the point, it should run on every state EXCEPT on the "init" and "exit" states because in those there is no content to show...

Reply:
Ah, k I understand. 😄

Reply:
For some reason it broke the functionality.. 🤔 But if I understand correctly if I don't add that code it just gets run 2 times to many during the application lifetime right?

Reply:
Yep, the code will get executed also in the login and the logout pages